azure-pipelines.yml

```diff
@@ -84,6 +84,25 @@ variables:
     ${{ else }}:
       value: OSX.13.Amd64
 
+parameters:
```

Comment:
Why did you use `paramaters` here and not just add onto `variables`? Usually I think of `parameters` in a top level YML as items we pass during manually queueing. This pipeline isn't manually queued that I'm aware of though.

Reply:
I don't believe you can have object types variables - using parameters allows me to just pass around the whole pool object everywhere